src/main/java/ir/sahab/kafkaconsumer/OffsetTracker.java

```diff
@@ -56,17 +64,29 @@
      *                                 partition, next calls to {@link #ack(int, long)} on that
      *                                 partition fails and returns false.
      */
-    public OffsetTracker(int pageSize, int maxOpenPagesPerPartition) {
+    public OffsetTracker(int pageSize, int maxOpenPagesPerPartition, MetricRegistry metricRegistry) {
```

Comment:
This constructor is not called from SmartCommitKafkaConsumer!

Reply:
Done